libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKUpgradeManager.java

```diff
@@ -248,4 +272,27 @@ private void upgradeFromVersions9_2_0Thru10_1_1To10_1_1PasscodeFixes() {
             }
         }
     }
+
+    // TODO: Remove upgrade step in Mobile SDK 12.0
+    private void updateFromBefore11_0_0() {
+        final LegacyAdminSettingsManager legacySettingsManager = new LegacyAdminSettingsManager();
+        final AdminSettingsManager settingsManager = SalesforceSDKManager.getInstance().getAdminSettingsManager();
+        final List<UserAccount> accounts = userManager.getAuthenticatedUsers();
+        if (accounts != null) {
+            for (UserAccount account : accounts) {
```

Comment:
During upgrade, we add the attributes found in the org level prefs to the user level prefs and then (at the end) clear the org level prefs